Packs/ClarotyXDome/IncidentFields/incidentfield-Claroty_xDome_Device_Insecure_Protocols.json

```diff
@@ -0,0 +1,32 @@
+{
+    "id": "incident_clarotyxdomedeviceinsecureprotocols",
+    "version": -1,
+    "modified": "2024-04-18T10:35:36.346255144Z",
+    "name": "Claroty xDome Device Insecure Protocols",
+    "ownerOnly": false,
+    "description": "The calculated level of the device’s ‘insecure protocols’ likelihood factor, such as \"Critical\", or \"High\"",
+    "cliName": "clarotyxdomedeviceinsecureprotocols",
+    "type": "shortText",
```

Comment:
maybe this should be a multi-select?

Reply:
good point.
So I just committed changes to custom fields `Device Effective Likelihood Subscore`, `Device Impact Subscore`, `Device Insecure Protocols`, `Device Known Vulnerabilities`, `Device Likelihood Subscore`, `Device Risk Score`.
changed their type to `singleSelect` with `selectValues: ["Very Low", "Low", "Medium", "High", "Critical"]`